### PR TITLE
Add suffixes to typos from "red" to "rep"

### PR DESCRIPTION
**Pull request by int-y1**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -43058,7 +43108,18 @@ regenratet->regenerated
...
 regenration->regeneration
 regenrative->regenerative
-regession->regression
+regeress->regress
+regeressed->regressed
+regeresses->regresses
+regeressing->regressing
+regeression->regression
+regeressions->regressions
+regess->regress, recess,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:43117]: Perhaps `egress` too?

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -43222,15 +43288,26 @@ reigstered->registered
...
 reigstration->registration
+reigstrations->registrations
+reigstries->registries
+reigstry->registry
 reimplemenet->reimplement
 reimplementaion->reimplementation
 reimplementaions->reimplementations
 reimplemention->reimplementation
 reimplementions->reimplementations
+reimplent->reimplement
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:43299]: Applies to all verbal forms and `reimplentation` too:
```suggestion
reimplent->reimplement, reimplant,
```

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -43433,9 +43520,18 @@ relocateable->relocatable
...
+relpaced->replaced
 relpacement->replacement
+relpacements->replacements
+relpaces->replaces
+relpacing->replacing
 relpy->reply
 reltive->relative
+relvance->relevance
+relvancy->relevancy
+relvant->relevant
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:43533]: Perhaps `reliant` too, although **I** and **V** are not neighbours on my keyboard?

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -43453,14 +43549,20 @@ remaned->remained
...
+remanining->remaining
+remanins->remains
 remaped->remapped
 remaping->remapping
 rembember->remember
 rembembered->remembered
 rembembering->remembering
 rembembers->remembers
-rember->remember
+rember->remember, renumber,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:43562]: Perhaps `member` as well?

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -43928,25 +44048,38 @@ repesentational->representational
 repesented->represented
 repesenting->representing
 repesents->represents
-repet->repeat
+repet->repeat, repent, repel,
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:44051]: Perhaps `reset` too?

File: codespell_lib/data/dictionary.txt (hunk 3 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:43533]: I couldn't find this typo.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (int-y1) [reply codespell_lib/data/dictionary.txt:43117]: I couldn't find this typo.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Pull request merged**